src/Text/Pandoc/Writers/Typst.hs

```diff
@@ -279,10 +279,17 @@ inlineToTypst inline =
                    then mempty
                    else nowrap $ brackets contents
     Image (_,_,kvs) _inlines (src,_tit) -> do
-      let width' = maybe mempty ((", width: " <>) . literal) $ lookup "width" kvs
-      let height' = maybe mempty ((", height: " <>) . literal) $
-                    lookup "height" kvs
-      return $ "#image(" <> doubleQuoted src <> width' <> height' <> ")"
+      let widthV = lookup "width" kvs
+      let heightV = lookup "height" kvs
+      let boxOpen = case (widthV, heightV) of
```

Comment:
Maybe this could reduce code clutter?
```haskell 
let (boxOpen, boxClose) = case (widthV, heightV) of
                                             (Nothing, Nothing) -> (mempty, mempty)
                                             _                              -> ("box(", ")")
```

Reply:
Also, just to make sure I'm understanding... if only width or only height is specified, it should still be boxed correct? Or do both need to be? If the latter, you're missing just a few cases

Reply:
> Also, just to make sure I'm understanding... if only width or only height is specified, it should still be boxed correct? Or do both need to be? If the latter, you're missing just a few cases

Looking at the tests, it appears it's the former, so disregard :)